src/Rairlie/LockingSession/SessionManager.php

```diff
@@ -19,17 +19,17 @@ protected function buildSession($handler)
 
         if ($this->app['config']['session.encrypt']) {
```

Comment:
```suggestion
        if ($this->container['config']['session.encrypt']) {
```

We want to change this one too yeah?

Reply:
Also what about LockingSessionServiceProvider.php ?